.github/workflows/issue_labeler.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Issue Labeling"
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  label-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            if (context.payload.issue.title.toLowerCase().includes("mrtk3") || context.payload.issue.body.toLowerCase().includes("mrtk3"))
```

Comment:
Do we want to add the equivalent logic for MRTK2?

Reply:
Sure!

Reply:
Done